### PR TITLE
Add code-remote-base to c# and vb content type definitions

### PR DESCRIPTION
**Pull request by dibarbet**
This tells the LSP client that it should activate on these content types.  How did this work before?
1.  Intellicode adds this base content type to the C# content type - https://devdiv.visualstudio.com/DevDiv/_git/IntelliCode-VS?path=%2Fsrc%2FVSIX%2FIntelliCode.VSIX%2FRefactorings%2FContentTypeDefinitions.cs&_a=contents&version=GBmain
2. On workspace load, we load our server ourselves.  This allows cntrl+Q to see the roslyn LSP server in C#/VB projects.
3. Any other local LSP feature that works off of the buffer in VB did not work.

Fixes https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1302281/

### CODE REVIEW

**Review by CyrusNajmabadi — APPROVED** [review]
i don't know waht this means.

File: src/EditorFeatures/VisualBasic/ContentType/ContentTypeDefinitions.vb [hunk 1]
```diff
@@ -11,10 +11,14 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.ContentType
 
         ''' <summary>
         ''' Definition of the primary VB content type.
+        ''' Also adds the LSP base content type To ensure the LSP client activates On C# files.
+        ''' From Microsoft.VisualStudio.LanguageServer.Client.CodeRemoteContentDefinition.CodeRemoteBaseTypeName
+        ''' We cannot directly reference the LSP client package in EditorFeatures as it Is a VS dependency.
```
Comment (dibarbet) [inline src/EditorFeatures/VisualBasic/ContentType/ContentTypeDefinitions.vb:16]:
```suggestion
        ''' Also adds the LSP base content type to ensure the LSP client activates On VB files.
        ''' From Microsoft.VisualStudio.LanguageServer.Client.CodeRemoteContentDefinition.CodeRemoteBaseTypeName
        ''' We cannot directly reference the LSP client package in EditorFeatures as it is a VS dependency.
```

**Review by davidwengier — APPROVED** [review]

File: src/EditorFeatures/CSharp/ContentType/ContentTypeDefinitions.cs [hunk 2]
```diff
@@ -17,6 +17,10 @@ internal static class ContentTypeDefinitions
         [Export]
         [Name(ContentTypeNames.CSharpContentType)]
         [BaseDefinition(ContentTypeNames.RoslynContentType)]
+        // Adds the LSP base content type to ensure the LSP client activates on C# files.
+        // From Microsoft.VisualStudio.LanguageServer.Client.CodeRemoteContentDefinition.CodeRemoteBaseTypeName
+        // We cannot directly reference the LSP client package in EditorFeatures as it is a VS dependency.
+        [BaseDefinition("code-languageserver-base")]
```
Comment (NTaylorMullen) [inline src/EditorFeatures/CSharp/ContentType/ContentTypeDefinitions.cs:23]: Uhhhh ya how did this work before lmao

Comment (dibarbet) [reply src/EditorFeatures/CSharp/ContentType/ContentTypeDefinitions.cs:23]: Intellicode!
https://devdiv.visualstudio.com/DevDiv/_git/IntelliCode-VS?path=%2Fsrc%2FVSIX%2FIntelliCode.VSIX%2FRefactorings%2FContentTypeDefinitions.cs&_a=contents&version=GBmain

Comment (NTaylorMullen) [reply src/EditorFeatures/CSharp/ContentType/ContentTypeDefinitions.cs:23]: lol the hellllllll

**Pull request merged**